src/foregroundTimeTracker.ts

```diff
@@ -9,7 +9,7 @@ export default class ForegroundTimeTracker {
     public totalTime: number = 0;
 
     constructor(timerKey: string) {
-        this.localStorageName = `mp-time-${timerKey}`;
+        this.localStorageName = `mprtcl-tos-${timerKey}`;
```

Comment:
Let's create a work item to create a local storage prefix constant or generator.